### ISSUE TITLE
[RFC] New command: dokku service:enter SERVICE

### ISSUE THREAD

**Issue by josegonzalez** [dokku/dokku-redis#65]
Currently if a user wants to run a command against a service container, they have to remember a weird incantation:

```
docker exec -it $(dokku postgres:info SERVICE --id) bash -c "$COMMAND"
```

Such lulz. Instead, it would be great to provide a command like this:

```
dokku postgres:enter SERVICE "$COMMAND"
```

We use `enter` instead of `run` as the command name as the semantics of `run` in dokku core mean spinning up a new container. `run` can be an alias for `enter`, with an added warning - to stderr - saying that the running container is the one where the command will be executed against.

We should also potentially add a warning to stderr saying that the destroying/recreating the container will result in a loss of the output of that command, in the case that the command changes the filesystem in some way that isn't saved to mounted directories (`data` or `config`). Users should expect to execute said commands again in the case that the container is brought up from scratch, though restarts should be just fine.

**Comment by josegonzalez** [comment]
I decided that adding a `run` command is a bit premature. We can't get rid of code - that makes workflows sad - and I'm not sold on adding built-ins in this way.

**Comment by josegonzalez** [comment]
Closed by d23b521142ceb74a50284dc5863702d52f9cddd6.
